docs/shared-content-source/docs/modules/develop/pages/use-spark-streamlets.adoc

```diff
@@ -28,6 +28,26 @@ codec of the data they manipulate.
 Currently, Cloudflow supports Avro and hence the classes are named `AvroInlet` and `AvroOutlet`.
 Each outlet also allows the user to define a partitioning function that will be used to partition the data.
 
+== Usage of different encodings in Spark streamlet
+
+Currently Cloudflow supports two different data encoding types - Avro and Protobuf. Both can be used for Spark streamlet,
+but you should should remember several rules. Protobuf support is based on https://scalapb.github.io/docs/sparksql/[Scala-PB Spark],
+which comes with its own `implicits` - `scalapb.spark.Implicits._` which is not compatible with `cloudflow.spark.sql.SQLImplicits._`.
+When using these `implicits` the following rules have to be observed:
+
+* If protobuf's encoding is not used, always use `cloudflow.spark.sql.SQLImplicits._`.
```

Comment:
Can't we use `org.apache.spark.sql.SQLImplicits` instead?

Reply:
(You might have told me before but I can't remember why we still have these)

Reply:
cloudflow.spark.sql.SQLImplicits and org.apache.spark.sql.SQLImplicits are identical. org.apache.spark.sql.SQLImplicits is used in conjunction with Spark session, see https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html 
Gerard created cloudflow.spark.sql.SQLImplicits so that it can be ised anywhere in the code.